source/JsMaterialX/JsMaterialXCore/JsInterface.cpp

```diff
@@ -9,19 +9,13 @@ namespace ems = emscripten;
 namespace mx = MaterialX;
 
 #define BIND_INTERFACE_TYPE_INSTANCE(NAME, T)                                 \
-.function("setParameterValue" #NAME, &mx::InterfaceElement::setParameterValue<T>) \
 .function("setInputValue" #NAME, &mx::InterfaceElement::setInputValue<T>)
 
 extern "C"
 {
     EMSCRIPTEN_BINDINGS(interface)
     {
 
-        ems::class_<mx::Parameter, ems::base<mx::ValueElement>>("Parameter")
```

Comment:
Remove parameter bindings from Javascript since it is not currently officially released yet, there is no need to add in something which is obseleted.